cinema_service/urls.py

```diff
@@ -1,8 +1,10 @@
 from django.contrib import admin
 from django.urls import path, include
+from user.urls import app_name
```

Comment:
The import of `app_name` from `user.urls` seems unnecessary unless it is explicitly used in the `namespace` argument. Ensure that `app_name` is defined in `user.urls` and is intended to be used as a namespace. Otherwise, consider removing this import.